### ISSUE TITLE
IDE0066 doesn't trigger when multiple cases map to default

### ISSUE THREAD

**Issue by stephentoub** [dotnet/roslyn#37947]
**Version Used**: 
3.3.0-beta3-19406-05+a1905991543bed104f7f7f0842aca2b65d263b87

**Steps to Reproduce**:
```C#
using System;

class Program
{
    public static void Main() { }

    public static string FromDay(DayOfWeek dayOfWeek)
    {
        switch (dayOfWeek)
        {
            case DayOfWeek.Monday:
                return "Monday";

            case DayOfWeek.Friday:
            default:
                return "Other";
        }
    }
}
```

**Expected Behavior**:
IDE0066 fires and refactors the switch into:
```C#
return dayOfWeek switch
{
    DayOfWeek.Monday => "Monday",
    _ => "Other",
};
```
If desired, the other cases could be added as comments, e.g.
```C#
return dayOfWeek switch
{
    DayOfWeek.Monday => "Monday",
    // DayOfWeek.Friday =>
    _ => "Other",
};
```

**Actual Behavior**:
IDE0066 does not trigger and does not offer to refactor.

**Comment by jasonmalinowski** [comment]
Moving out of 16.3 -- at this point not going to worry about "this could work, but doesn't".
